keyboard/src/lib/y2keyboard/keyboards.rb

```diff
@@ -42,226 +42,286 @@ def self.all_keyboards
 
   # @see all_keyboards
   def self.always_present_keyboards
+    # Now (2021-12-07) using keymaps from the new kbd package
+    # from /usr/share/kbd/keymaps/xkb, no longer kbd-legacy
+    # from /usr/share/kbd/keymaps/{amiga,atari,i386,include,mac,sun}
+    #
+    # "not_in_xkb": the xkb layout has no Latin letters and is therefore
+    # excluded from the console conversion. Dropping kbd-legacy would break
+    # these languages.
+    #
+    # "us_symlink": it has always been just a symlink to the us layout.
+    # bsc#1194609 moves the symlinks to kbd.rpm
+    #
+    # See also  man xkeyboard-config
     [
       { "description" => _("English (US)"),
         "alias" => "english-us",
         "code" => "us",
+        # No different legacy_code
         "suggested_for_lang" => ["ar_eg", "en", "nl_BE"]
       },
       { "description" => _("English (UK)"),
         "alias" => "english-uk",
-        "code" => "uk"
+        "code" => "gb",
+        "legacy_code" => "uk"
```

Comment:
new legacy_code key documentation is missing at https://github.com/yast/yast-country/pull/288/files#diff-c14578e93ba62536ac66ff328b6b6c18b8a76b064eb5936288b01cb1aa093e0bR37

Reply:
Added doc.